### ISSUE TITLE
Wrong comment 0.01

### ISSUE THREAD

**Issue by off99555** [lllyasviel/ControlNet#392]
https://github.com/lllyasviel/ControlNet/blob/d3284fcd0972c510635a4f5abe2eeb71dc0de524/gradio_canny2image.py#L52

it should be 0.1, right?

**Comment by lllyasviel** [comment]
yes it seems that it should be 0.825**12<0.01**0.5 - will fix soon
